rust_snuba/src/processors/replays.rs

```diff
@@ -357,6 +368,12 @@ struct ReplayEventLinkEvent {
     warning_id: Option<Uuid>,
 }
 
+#[derive(Debug, Deserialize)]
+struct ReplayViewedEvent {
+    user_id: u64, // not the same as the column in ReplayRow. This is the id of a Sentry user in postgres, belonging to the authorized user POSTing to the viewed-by endpoint
```

Comment:
Rather than putting a code comment here let's name the field something more contextual so it doesn't require one.  We could call it `viewed_by_id` or `sentry_user_id`.  Naming it `user_id` will create confusion because of the `user_id` column.